migrations/11-enum-for-user-state.yml

```diff
@@ -0,0 +1,16 @@
+up:
+  - |
+    UPDATE users
+    SET user_state = 'inactive'
+    WHERE user_state not in (VALUES ('member'),('inactive'),('administrator'),('moderator'))
+  - CREATE TYPE enum_member_state AS ENUM ('inactive','member', 'moderator', 'administrator')
```

Comment:
Thanks a lot for your contribution. 
It is almost perfect, expect that https://github.com/xvw/muhokama/runs/8303873016?check_suite_focus=true fail because `administrator` was referenced as `admin`. So I suggest this (in order to reduce the diff and the complexity of the update quiry):

```diff
- WHERE user_state not in (VALUES ('member'),('inactive'),('administrator'),('moderator'))
+ WHERE user_state not in (VALUES ('member'),('inactive'),('admin'),('moderator'))
- CREATE TYPE enum_member_state AS ENUM ('inactive','member', 'moderator', 'administrator')
+ CREATE TYPE enum_member_state AS ENUM ('inactive','member', 'moderator', 'admin')
```

Reply:
(It should be sufficient)